tracer/test/Datadog.Trace.ClrProfiler.IntegrationTests/CosmosTests.cs

```diff
@@ -34,13 +33,12 @@ public static IEnumerable<object[]> GetEnabledConfig()
 
         public override Result ValidateIntegrationSpan(MockSpan span, string metadataSchemaVersion) => span.IsCosmosDb(metadataSchemaVersion);
 
-        [SkippableTheory]
+        [Theory]
         [MemberData(nameof(GetEnabledConfig))]
         [Trait("Category", "EndToEnd")]
         [Trait("RunOnWindows", "True")]
```

Comment:
So I think the tests are still meant to run only on Windows, but the linux docker container means it could run on all platforms?

I wasn't sure exactly what we want to do here

Reply:
Yeah, this gives a big flip. Currently the tests assume we're using the powershell-started cosmos emulator, so can only run on Windows. With this change, we can _only_ run on Linux, so we need to remove the `RunOnWindows` config for now.

Also, you haven't configured the docker-compose for arm64 yet (there's separate testing and start-dependencies stages), so we'll need to do that.

```suggestion
```